src/main/resources/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "platformName": "android",
   "isRemote": false,
-  "remoteConnectionUrl": "http://127.0.0.1:4723/wd/hub",
+  "remoteConnectionUrl": "http://127.0.0.1:4723",
```

Comment:
am I correct, that the users of lower versions will have to make similar changes in theirs configs?
if so, don't forget to add this into release notes.

Reply:
It actually depends on appium server version that you have installed.
Appium v.1 requires "/wd/hub" in address, while v.2 doesn't have it by default (but is not released yet)
I would revert these changes, end users should be self-aware of it when migrating to v.2

Reply:
Fixed, all tests have passed on both remote and local configurations. Changes have been made to the pipeline script, so that remote URL conforms with an old configuration (same as described here: https://github.com/sudharsan-selvaraj/appium-dashboard-plugin/blob/987f89f14fee83ccb3d610b9457b1af66719fef0/README.md#troubleshooting )

Reply:
Added info to README.md